index.js

```diff
@@ -1,3 +1,6 @@
+console.clear();
+import { createCharacterCard } from "./components/card/card.js";
```

Comment:
As the export from this function is going to be a `default export`, you need to adjust the import as well:

```suggestion
import createCharacterCard from "./components/card/card.js";
```